lib/package-manager.coffee

```diff
@@ -4,6 +4,8 @@ _ = require 'underscore-plus'
 Q = require 'q'
 semver = require 'semver'
 url = require 'url'
+apm = require 'apm'
```

Comment:
I'd like to keep the settings view only interacting with apm via an external process so we can ensure apm exposes a useful enough API to be used.

What do you think about making this flow possible from an `apm` command such as something like `apm view --json --compatible 0.188.0` or somethign?